src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__container">
+        <div class="stopwatch__minute"></div>
+        <div class="stopwatch__seconds"></div>
```

Comment:
The class name `stopwatch__seconds` seems to be a typo. According to BEM methodology and common naming conventions, it should be `stopwatch__second` to match singular form like `minute` and `center`.